cmd/loadTests.go

```diff
@@ -1261,11 +1248,11 @@ func (h ItsSuccessHandler) HandleSuccess(ctx *JourneyContext, itsName string, ti
 	handleItsSuccess(ctx, itsName, h.Username, h.Application, timeInSeconds)
 }
 
-type CdqSuccessHandler struct{}
+// type CdqSuccessHandler struct{}
```

Comment:
Leftover?

Reply:
Hello. `cmd/loadTests.go` will be deleted completely in https://github.com/redhat-appstudio/e2e-tests/pull/1163 anyway, so feel free to ignore I would say.

Reply:
Removed.